From 065d74acdd04618eb7e059cf472a22a1a90ea5ae Mon Sep 17 00:00:00 2001
From: Hao Wu <hao.wu@terranet.se>
Date: Fri, 3 Aug 2018 09:23:17 +0200
Subject: [PATCH] modify Makefile to fit yocto only

Change-Id: I0d23758d88d428d78d501e9a433d2f0a0f0e8dfc
---
 Makefile | 28 ++++++----------------------
 1 file changed, 6 insertions(+), 22 deletions(-)

diff --git a/Makefile b/Makefile
index fc50648..8eacf07 100644
--- a/Makefile
+++ b/Makefile
@@ -1,21 +1,8 @@
 PWD := $(shell pwd)
-ifeq ($(CROSS_COMPILE),)
 SHELL := /bin/sh
-CC = gcc
-KVER  := $(shell uname -r)
-KSRC := /lib/modules/$(KVER)/build
-# Handle the move of the entire rtlwifi tree
-ifneq ("","$(wildcard /lib/modules/$(KVER)/kernel/drivers/net/wireless/realtek)")
-MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/realtek/rtlwifi
-else
-MODDESTDIR := /lib/modules/$(KVER)/kernel/drivers/net/wireless/rtlwifi
-endif
-else
-KSRC := $(PWD)/../linux-imx
-#KSRC := $(PWD)/../terranet_linux-imx/linux-imx
-MODDESTDIR := $(KSRC)/drivers/net/wireless/rtlwifi
-endif
-FIRMWAREDIR := /lib/firmware/
+#MODDESTDIR := $(KSRC)/drivers/net/wireless/rtlwifi
+#FIRMWAREDIR := /lib/firmware/
+#FIRMWAREDIR := $(KERNEL_SRC)/firmware
 CLR_MODULE_FILES := *.mod.c *.mod *.o .*.cmd *.ko *~ .tmp_versions* modules.order Module.symvers
 SYMBOL_FILE := Module.symvers
 #Handle the compression option for modules in 3.18+
@@ -95,17 +82,14 @@ ccflags-y += -D__CHECK_ENDIAN__
 ccflags-y += -DCONFIG_RTLWIFI_DEBUG
 subdir-ccflags-y += -DCONFIG_RTLWIFI_DEBUG
 
-ifeq ("$(KVER)", "3.14.35-031435-generic")
-ccflags-y += -D_ieee80211_is_robust_mgmt_frame=ieee80211_is_robust_mgmt_frame
-subdir-ccflags-y += -D_ieee80211_is_robust_mgmt_frame=ieee80211_is_robust_mgmt_frame
-endif
-
 CHECKFLAGS += -D__CHECK_ENDIAN__
 
 all:
 	$(MAKE) -C $(KSRC) M=$(PWD) modules
-install: all
+#install: all
+modules_install:
 	$(MAKE) -C $(KSRC) M=$(PWD) modules_install
+
 # ifeq (,$(wildcard ./backup_drivers.tar))
 # 	@echo Making backups
 # 	@tar cPf backup_drivers.tar $(MODDESTDIR)
-- 
2.7.4

